content-services/5.2/using/share.md

```diff
@@ -1,297 +1,129 @@
+* **Admin Tools**: System administrators only have this option which opens the Admin Tools.
+* **Sites Manager**: Only Alfresco administrators and Site administrators have this option which opens the Sites Manager. If you are an Alfresco administrator, you can access the Site Manager through the **Admin Tools** on the toolbar. If you are a Site administrator, you'll have an additional **Sites Manager** option on the toolbar.
+* **`User` menu**: Click on your name to open the user menu. You can access your user profile, change your password, open the user help, and log out.
+* **Search**: Use the search box to find files, sites, and people.
 
-    Click on your name to open the user menu. You can access your user profile, change your password, open the user help, and log out.
-
--   **Search**
-
-    Use the search box to find files, sites, and people.
-
-### User dashboard {#user-dashboard}
+## User dashboard
 
 Wherever you are in Alfresco Share, you can click **Home** or **User Dashboard** on your user menu to go to your dashboard.
 
 The dashlets on this page give you a simple and up-to-date overview of what's been going on in the sites you're a member of.
 
-> **Important:** If you've [set your home page]({% link content-services/5.2/using/dashboard.md %}#setting-your-home-page) then when you click **Home** that page will open.
-
-You can customize your dashboard to display the information that interests you. Each dashlet you select is displayed on the dashboard. The available dashlets are:
-
--   **My Sites**
-
-    Displays the sites that you belong to
-
--   **My Activities**
-
-    Tracks the most recent activities in your sites
-
--   **My Tasks**
-
-    Displays your tasks
-
--   **My Documents**
-
-    Tracks your content in all site libraries
-
--   **My Profile**
-
-    Displays a summary of your personal details
-
--   **My Calendar**
-
-    Displays upcoming events scheduled for your sites
-
--   **Content I'm Editing**
-
-    Displays the last three library items, blog posts (drafts, not published posts), wiki pages, and discussion forum posts that you edited
-
--   **Web View**
-
-    Display a website which you select by hovering over the dashlet on your dashboard
-
--   **RSS Feed**
-
-    Hover over the dashlet on your dashboard to select an RSS feed (displays the Alfresco website feed by default)
-
--   **Alfresco Add-ons RSS Feed**
-
-    Displays the latest news from the Alfresco Add-ons website by default
-
--   **My Discussions**
-
-    Displays the most recent topics created in the discussion forums of all sites you belong to
-
--   **Site Search**
-
-    Lets you quickly search all sites you belong to
-
--   **Saved Search**
-
-    Displays the results of a specific search that you configure
-
--   **My Document Workspaces**
+> **Note:** If you've [set your home page]({% link content-services/5.2/using/dashboard.md %}#sethomepage) then when you click **Home** that page will open.

```

Comment:
Incorrect in-page link

Reply:
Fixed the related link in `content-services/5.2/using/dashboard.md`